lib/schema/schema.js

```diff
@@ -51,6 +51,14 @@ const VendorType = new GraphQLObjectType({
           .join('products', { 'vendors.id': 'products.vendor_id'}) //check this line
           .where('products.vendor_id', parent.id)
       }
+    },
+    market_vendors: {
+      type: GraphQLList(MarketVendorType),
+      resolve(parent, args) {
+        return database('vendors')
+          .join('market_vendors', { 'vendors.id': 'market_vendors.market_id'})
+          .where('market_vendors.market_id', parent.id)
+      }
```

Comment:
We need the `markets` instead of `market_vendors` here. You'll need an additional join with the `markets` table.